### ISSUE TITLE
Fix latest release info on Wikipedia

### ISSUE THREAD

**Issue by PanderMusubi** [armbian/documentation#394]
On https://en.wikipedia.org/wiki/Armbian currently the latest release reads:

    23.11[[1]](https://en.wikipedia.org/wiki/Armbian#cite_note-wikidata-5c55f50e6831912d5cbb06a0b48fb2ab584b1ddf-v9-1) / 29 November 2023; 1 January 2024; Error: first parameter cannot be parsed as a date or time.

The error displayed needs to be fixed.

**Comment by igorpecovnik** [comment]
It was even flagged for deletion. https://en.wikipedia.org/wiki/Wikipedia:Articles_for_deletion/Armbian

I have no power to fight this too.

**Comment by PanderMusubi** [comment]
I con help a bit, but regarding this issue I have no idea how go about it.

**Comment by igorpecovnik** [comment]
> I con help a bit, but regarding this issue I have no idea how go about it.

It seems issue is closed as they find relevant sources. Here https://armbian.atlassian.net/browse/AR-2044 we have our plan, but this was eventually removed, so we need someone that understands how proper Wikipedia stub should look like. Doing all the work again and again be removed, makes no sense.

**Comment by PanderMusubi** [comment]
Meanwhile I added useful information with preoper references https://en.wikipedia.org/wiki/Armbian

**Comment by EvilOlaf** [comment]
Thanks.

24.2.0 will not feature Trixie or Noble as main userspaces but Bookworm and Jammy as current stable LTS userspaces from Debian and Ubuntu.
Either Noble will be bumped to stable afterwards with a minor update (24.2.2 or something like this) or be featured in 24.05. Trixie on the other hand still has lots of time to go IIRC.

**Comment by viraniac** [comment]
> I can help a bit, but regarding this issue, I have no idea how go about it.

I tried taking a look, but my IP seems to be blocked for editing. So can't really try to figure out or fix the same. If someone can share the raw markup for it, I can try to figure it out

**Comment by viraniac** [comment]
> Question, Is this correct architecture? https://www.armbian.com/download/?tx_maker=libre-computer&arch=armhf as all other Libre copmuter SBCs are aarch64

Yes, thats correct. Its based on Allwinner H3 which is a 32bit SoC.

**Comment by viraniac** [comment]
> See https://pastebin.com/yRzyYpQE for markup.

Thanks, will try it out in a mediawiki docker container and will try to fix the same

**Comment by viraniac** [comment]
ok, looked into the markup. I think I will need to know what P348 is to fix the issue. Its not present in the markup

**Comment by viraniac** [comment]
Found P577 at https://www.wikidata.org/wiki/Q25025335. Its the published date field of 23.11.2. Can you try changing it to "January 1, 2024" i.e. move the date and month around and add a comma after the date

**Comment by PanderMusubi** [comment]
Can you give me exact examples to try?

**Comment by viraniac** [comment]
Click on edit button shown below

<img width="868" alt="Screenshot 2024-02-06 at 9 44 14 PM" src="https://github.com/armbian/documentation/assets/6690664/240fea50-ff8d-4834-8820-21968bda71db">
 
and change the publication date to value I suggested above.

We probably also need to change
```
| latest release date = {{start date and age|{{wikidata|qualifier|P348|P577}}}}
```
to
```
| latest release date = {{Time ago|{{wikidata|qualifier|P348|P577}}}}
```

**Comment by viraniac** [comment]
its possible that only changing to Time ago itself might solve the issue

**Comment by igorpecovnik** [comment]
Some sources to add:
https://www.tomshardware.com/news/raspberry-pi-64-bit-armbian-new-release
https://openpeerpower.io/docs/installation/armbian/
https://alternativeto.net/software/armbian/about/
https://nlnet.nl/project/Armbian/
https://www.linux-magazine.com/Online/News/Armbian-23.05-Now-Available
https://girishjoshi.io/post/install-armbian-on-orange-pi-zero/
https://docs.jethome.ru/en/controllers/linux/software/armbian.html
https://linuxhint.com/raspberry-pi-os-vs-armbian-vs-debian-gnu-linux/
http://kofa.mmto.arizona.edu/orange_pi/armbian/explore/
https://wiki.debian.org/Derivatives/Census/Armbian
https://jobcespedes.dev/2023/11/building-a-k3s-cluster-with-armbian-on-orange-pi-5-plus/
https://www.hackster.io/pradeeplogu0/experience-the-armbian-with-lattepanda-3-delta-e57640
https://www.youtube.com/watch?v=Fg966ivZlrc
https://longervision.github.io/2019/05/12/SBCs/ARM/orange-pi-3-armbian/
https://www.youtube.com/watch?v=IBZ3-bEsqkA (and many more from NicoD)
https://www.youtube.com/watch?v=DzzDzQ3hCdI
https://www.youtube.com/watch?v=_7jQrXn8Yb0
https://www.youtube.com/watch?v=T359Bet3zYw
https://www.youtube.com/watch?v=3-M3viI5n-Y
https://www.hackster.io/news/giuseppe-it9ybg-grasso-saves-an-old-android-set-top-box-from-landfill-and-puts-an-sdr-online-bf911a476adc
https://www.theregister.com/2023/09/08/linux_on_the_thinkpad_x13s/
https://www.hackster.io/news/wlan-pi-2-0-now-built-on-armbian-turns-your-raspberry-pi-into-a-network-performance-analyzer-5311ba0c1aeb
https://www.cnx-software.com/2019/06/22/create-minimal-debian-images-debos-armbian/
https://www.cnx-software.com/2022/01/24/building-nanopi-m4v2-all-in-one-linux-pc-running-armbian-ubuntu-debian/
https://www.hackster.io/news/hands-on-with-the-kobol-helios64-a-five-bay-armbian-based-rk3399-powered-open-spec-nas-system-1b4defc9c680
https://www.cnx-software.com/2022/08/23/29-inovato-quadra-arm-mini-pc-runs-armbian-on-allwinner-h6-processor/
https://www.computerbase.de/2021-05/armbian-21.05-ubuntu-distribution-fuer-einplatinencomputer/

**Comment by igorpecovnik** [comment]
Updated, closing.
